### DISCUSSION TITLE
Add support for `test_active_file`

### DISCUSSION THREAD

**Discussion by ilyaZar** [REditorSupport/vscode-R#1777] [Ideas]
**Problem**
A convenient way for continuous testing (in RStudio) is to only run `devtools::test_active_file()` to test only the active test-file. 

This is especially beneficial, if the project is large and a blunt `testthat::test()` takes too much time. Currently, `devtools::test_active_file()` gives
```r
r$> devtools::test_active_file()
Error in rstudioapi::executeCommand("activateConsole", quiet = TRUE) : 
  This {rstudioapi} function is not currently implemented for VSCode.
```

**Solution**
I assume you folks do not have much time, but I am happy to work on this, if:
- it is possible with moderate difficulty
- give me some hints on where to start e.g. other similar commands already ported from the `{rstudioapi}` so I can take their implementation as a blueprint

The project here is new to me, but if you give me some guidance on where to start a possible PR I'd happy to dive in! 

Cheers and thanks for this extension, it's a gem !

**Comment by ThomasSoeiro** [comment]
@eitsupi @Fred-Wu @grantmcdermott Can we close in favour of https://github.com/REditorSupport/vscode-R/issues/59? (If so, I will add a comment in https://github.com/REditorSupport/vscode-R/issues/59 to specifically mention `devtools::test_active_file()`.)
